Lesson_4js_homework/js_4/script.js

```diff
@@ -0,0 +1,123 @@
+//ZAD
+let java = prompt("Какое «официальное» название JavaScript?");
+if (java == `ECMAScript`) {
+  alert(`True`);
+} else {
+  alert(`Не знаете? ECMAScript!`);
+}
+
+//ZAD
+let num = +prompt("Enter number:");
+if (num > 0) {
+  alert(1);
+} else if (num < 0) {
+  alert(-1);
+} else {
+  alert(0);
+}
+
+//ZAD
+let age = prompt("Enter your age:");
+if (age >= 14 && age <= 90) {
+  alert(``);
+}
+
+//ZAD
+let age = prompt("Enter your age:"); //НЕ находится в диапазоне 14 и 90 включительно
+if (!(age >= 14 && age <= 90)) {
+}
+
+if (age < 14 || age > 90); //НЕ находится в диапазоне 14 и 90 включительно. без этого оператора"!"
+
+//ZAD
+let password = prompt("Enter your password:");
+
+if (password == `Admin`) {
+  let pass = prompt("Пароль?");
+  if (pass == "Я главный") {
+    alert(`Здравствуйте!`);
+  } else if (pass == " " || pass == null) {
+    alert(`Отменено`);
+  } else {
+    alert(`Неверный пароль`);
+  }
+} else if (password == " " || password == null) {
+  alert(`'Отменено'`);
+} else {
+  alert(`Я вас не знаю`);
+}
+
+//HOMEWORK
+//Запросить у пользователя его возраст и определить кем он явдяется...
+
+let age= +prompt (" Enter your age:");
+
+if ( age>0 || age=<2) {
+  alert ("A child");
+}else if (age>=12 || age<18){
+  alert ("A teenager");
+} else if (age>=18 || age<=60){
+  alert ("An adult");
+} else if (age>60){
+  alert ("A pensioner");
+}
```

Comment:
чтоб проверить, что возраст между 0 и 2, нужно использовать **&&**, то есть возраст больше 0 **И** одновременно меньше или равно 2
```suggestion
if ( age > 0 && age <= 2) {
  alert ("A child");
}else if (age >= 12 && age < 18){
  alert ("A teenager");
} else if (age >= 18 && age <= 60){
  alert ("An adult");
} else if (age > 60){
  alert ("A pensioner");
}
```